microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -147,6 +147,24 @@ The CIDR notation for grouping IP addresses was removed from the `clusterNetwork
 
 * Previously, certificate authorities generated `kubeconfig` files for {product-title}'s embedded components. With this update, certificate authorities are reconfigured to ensure the `kubeconfig` files are independent. (link:https://issues.redhat.com/browse/OCPBUGS-8301[*OCPBUGS-8301*])
 
+* Previously, the sos wrapper helper was not present and the sos command manually. With this update, the sos wrapper was added the {microshift-short} and you no longer need to run the sos command manually when debugging. (link:https://issues.redhat.com/browse/OCPBUGS-19772[*OCPBUGS-19772*])
+
+* Previously, {microshift-short} did not generate version details at startup. With this update, {microshift-short} now logs version details at startup. (link:https://issues.redhat.com/browse/OCPBUGS-19638[*OCPBUGS19638*])
+
+* Previously, node IP certificates were using the old IP instead of the new causing all external certificates not to verify against the new IP. With this update, certificates update when the node IP changes. (link:https://issues.redhat.com/browse/OCPBUGS-19315[*OCPBUGS-19315*])
+
+* Previously, changing IP results left `kubeconfigs` for the old IP on disk making the `kubeconfigs` unusable because they target an IP that does not belong to {microshift-short} anymore. With this update, the PLD `kubeconfigs` are deleted when the IP changes. (link:https://issues.redhat.com/browse/OCPBUGS-15740[*OCPBUGS-15740*])
```

Comment:
```suggestion
* Previously, after changing the IP address, `kubeconfig` files for the previous IP remained. These `kubeconfig` files were unusable because they targeted an IP not used by {microshift-short}. With this update, the `kubeconfig` files associated with the previous IP address are deleted when the IP changes. (link:https://issues.redhat.com/browse/OCPBUGS-15740[*OCPBUGS-15740*])
```